src/modules/clusters/cluster-list/cluster-list.html

```diff
@@ -88,28 +88,28 @@ <h1>No Clusters Detected</h1>
                         <div class="list-view-pf-additional-info">
                             <div class="list-view-pf-additional-info-item list-view-pf-additional-info-item-stacked">
                                 <div class="bold-text">Cluster Version</div>
-                                <h5 class="pull-left counts">{{cluster.sdsName === 'gluster'? "RHGS": ""}} {{cluster.sdsVersion}}</h5>
+                                <h5 class="pull-left cluster-version">{{cluster.sdsName}} {{cluster.sdsVersion}}</h5>
```

Comment:
@cloudbehl, `uppercase` will make 'gluster' as 'GLUSTER', `text-transform: capitalize` will make 'gluster' as 'Gluster', hence only capitalize first letter. There is no per-defined angular filter for this.

Reply:
@a2batic Ohh ok I thought we were making the whole word capitalize. Sorry

If we want to just capitalize the first word then why are we even using the CSS class as we already have the capitalize[1] filter available in our code. We just need  {{ cluster.sdsName | capitalize }} and it's done.

Also, it will be of practice not to write the repetitive code in CSS files rather make a filter if and then use it. It makes the code look much cleaner.


[1].https://github.com/Tendrl/ui/blob/65690ee6d4d63361c5ec62b99ff7d970b25a5eb0/src/commons/filters/bytes.js

Reply:
See the example below. We are already using it in our code. 

https://github.com/Tendrl/ui/blob/d1162e00a02d1ef004e36b29effd2762e0ac2895/src/modules/volumes/volume-list/volume-list.html

Reply:
@gnehapk @a2batic There are places we are already using CSS please try removing them too.

https://github.com/Tendrl/ui/search?p=2&q=capitalize&type=&utf8=%E2%9C%93

Reply:
@cloudbehl, +1, that will make it easy. :+1: